### DISCUSSION TITLE
Publication by non-governmental institutions

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#53] [General]
Despite the good frame provided by the DCAT-AP to avoid difficulties in finding and re-using public information. There is still a gap in data accessibility, when the data provider is a non-governmental institution, like a private company or university. These institutions can provide very useful information, e.g.:  the distribution of a particular plant species in a forest. The issue comes from the need of providing a DCAT-AP endpoint, to be part of this highly useful initiative.

To allow the inclusion of this “alternative” data ensuring, at the same time, that the information provided is reliable and worth its incorporation in the catalogue, it is recommended to establish an autonomous body capable to provide the necessary infrastructure to alternative institutions at the time that ensures the quality of the data provided by them.

**Comment by sMorrone** [comment]
issue reported is linked to https://github.com/gopeg/issues-barriers/issues/3

**Comment by AndersOstmanNovogit** [comment]
This issue is linked to issue #27. It is reported in section 4.5.4 of D4.3.

**Reply by AndersOstmanNovogit** [reply]
Now published in D4.3 section 4.4.5
